docs/how_to/documenting_new_feature_toggles.rst

```diff
@@ -0,0 +1,74 @@
+.. _documenting_new_feature_toggles:
+
+=======================================
+How to: Documenting new feature toggles
+=======================================
+
+Feature toggles are boolean values that enable or disable the use of a certain feature. There are many of those across the Open edX codebase: in the form of Django settings, Configuration models, Waffle switches and flags, for example. It this important that these feature toggles be correctly annotated to address the following scenarios:
+
+- Generation of human-readable documentation of existing feature toggles for Open edX operators: instead of grepping a large codebase, operators (sysadmins and product owners, for instance) should be able to search an online document to learn how to use and enable features on their platforms.
+- Automated discovery and verification of feature toggles on a running platform: this allows, for instance, the comparison of feature toggles between different environments at runtime.
```

Comment:
This feels misleading.  The Toggle State Endpoint does not include code annotation data.  We have a separate script that joins the output of this endpoint with code annotation data. Not sure if and how to bake annotation data into the endpoint results?

Reply:
> The Toggle State Endpoint does not include code annotation data.

Right! I didn't realise that. I'll rephrase this sentence.